scripts/dev/check_parameters.php

```diff
@@ -243,34 +239,6 @@ function check_function($name, $txt, $offset)
 						}
 					break;
 
-					case '&':
```

Comment:
I *think* so!

Since these cases only apply to PHP 6 I pulled them out. We still get the error handling for unknown characters for the default case.